packages/tauri/webpack.config.js

```diff
@@ -65,7 +67,7 @@ module.exports = {
             meta: {
                 "Content-Security-Policy": {
                     "http-equiv": "Content-Security-Policy",
-                    content: `default-src 'self' ${serverUrl} https://api.pwnedpasswords.com blob:; style-src 'self' 'unsafe-inline'; object-src 'self' blob:; frame-src 'self'; img-src 'self' blob: data: https:;`,
+                    content: `default-src 'self' ${serverUrl} ${websocketServerUrl} https://api.pwnedpasswords.com blob:; style-src 'self' 'unsafe-inline'; object-src 'self' blob:; frame-src 'self'; img-src 'self' blob: data: https:;`,
```

Comment:
The app wouldn't start without this even because it was trying to connect to the websocket server. I'm not sure if this was something that recently changed, or if I missed something else. I spent way too much time getting Tauri updated to dig deeper into this part right now.